lua/nvim-tree/view.lua

```diff
@@ -385,6 +385,10 @@ function M._prevent_buffer_override()
     require("nvim-tree.renderer").draw()
     a.nvim_win_close(curwin, { force = true })
     require("nvim-tree.actions.open-file").fn("edit", bufname)
+
+    if require("nvim-tree").get_config().actions.open_file.quit_on_open then
```

Comment:
I like that this is a direct call instead of an au.

I will test over the weekend.

Reply:
Please follow the existing pattern and add to `M.View.quit_on_open`. It needs a cleanup, but Not Today.